lib/__tests__/conversion.test.ts

```diff
@@ -1,57 +1,57 @@
-import AlgoliaInsights from '../insights';
+import AlgoliaInsights from "../insights";
 
 const credentials = {
-  apiKey: 'test',
-  applicationID: 'test'
```

Comment:
I don't really care much, but it seems like the diff would be smaller if you use `singleQuote: true` in the prettier config

Reply:
you're right, let's try that

Reply:
I tried the singleQuote option of prettier, and I faced issues:
It does transform most of the strings to singleQuote, unless it results in to many escapes, in case which is falls back again to double quotes (Behaviour is explain [here](https://github.com/prettier/prettier/issues/2338#issuecomment-311776517)) 

In shorts: singleQuote:true leads prettier mixe quotes 😨 and tslint is not happy